scripts/create-pr.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+PR_TITLE="chore (automated): OpenSauced updates"
+BRANCH_NAME="chore_automated_open-sauced-updates_$(date +%s)"
+
+git branch $BRANCH_NAME
```

Comment:
The env var should be double quoted to prevent globbing / word splitting:

```suggestion
git branch "$BRANCH_NAME"
```